bitnami/postgresql-ha/templates/postgresql/statefulset.yaml

```diff
@@ -327,6 +327,20 @@ spec:
             - name: REPMGR_DATABASE
               value: {{ (include "postgresql-ha.repmgrDatabase" .) | quote }}
             {{- end }}
+            - name: REPMGR_FENCE_OLD_PRIMARY
+              value: {{ ternary "true" "false" .Values.postgresql.repmgrFenceOldPrimary | quote }}
```

Comment:
You would need to change this accordingly with my comments on https://github.com/bitnami/bitnami-docker-postgresql-repmgr/pull/112 .

Reply:
Hi I have changed the ternary for this accordingly as well